pkgs/games/anki/bin.nix

```diff
@@ -58,6 +58,7 @@ if stdenv.isLinux then buildFHSUserEnv (appimageTools.defaultFhsEnvArgs // {
   targetPkgs = pkgs: (with pkgs; [ xorg.libxkbfile krb5 ]);
 
   runScript = writeShellScript "anki-wrapper.sh" ''
+    export SSL_CERT_FILE=/etc/ssl/certs/ca-bundle.crt
```

Comment:
If we're patching this anyways, might as well patch in the Nix store path.

Reply:
Is there a way to do that such that the package continues to work correctly on non-NixOS hosts?

Reply:
The regular ca-bundle should still work on non-NixOS. It's just data.

Reply:
Hmm, it looks like if I use the path in cacert, it would ignore any configuration done in [ca.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/security/ca.nix). Then again, I guess it's not guaranteed that this path will exist on every Linux distro, so there will be issues regardless...

To respect NixOS/other distro configuration, perhaps we should set `SSL_CERT_FILE` conditionally if certain well-known file paths exist? For example, something like...

```sh
for i in /etc/ssl/certs/ca-bundle.crt \
           /etc/ssl/certs/ca-certificates.crt \
           /etc/pki/tls/certs/ca-bundle.crt \
           ${cacert}/etc/ssl/certs/ca-bundle.crt
do
  if test -f "$i"; then
    export SSL_CERT_FILE="$i"
    break
  fi
done

exec ${unpacked}/bin/anki
```

Reply:
Good points. How do other packages handle this?

Reply:
Judging by a search for `SSL_CERT_FILE`, it seems like most packages just use the path in `${cacert}`. I think it's probably fine for 99% of use cases to do that.

Reply:
Pushed a change that sets it if unset. That seems like a pretty good middle ground.